apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/internal/JsonRecordSerializer.kt

```diff
@@ -16,18 +17,30 @@ import okio.use
  */
 @ApolloInternal
 object JsonRecordSerializer {
+  private const val KEY_ARGUMENTS = "__arguments"
+  private const val KEY_METADATA = "__metadata"
```

Comment:
Let's use `__apm` in case https://github.com/graphql/graphql-spec/issues/300 lands one day and uses `__metadata` ?

(`ap` is for Apollo and `m` for metadata, since this is going to be written multiple times on disk, we might as well save a few bytes)

Reply:
Good call!